actors/miner/tests/util.rs

```diff
@@ -3010,15 +3168,50 @@ fn expect_compute_unsealed_cid_from_pieces(
     let expected_unsealed_cid_inputs: Vec<PieceInfo> =
         pieces.iter().map(|p| PieceInfo { size: p.size, cid: p.cid }).collect();
     let unsealed_cid = make_piece_cid(format!("unsealed-{}", sno).as_bytes());
-    rt.expect_compute_unsealed_sector_cid(
-        seal_proof_type,
-        expected_unsealed_cid_inputs,
-        unsealed_cid.clone(),
-        ExitCode::OK,
-    );
+    if !expected_unsealed_cid_inputs.is_empty() {
+        rt.expect_compute_unsealed_sector_cid(
+            seal_proof_type,
+            expected_unsealed_cid_inputs,
+            unsealed_cid.clone(),
+            ExitCode::OK,
+        );
+    }
     unsealed_cid
 }
 
+fn expect_validate_precommits(
```

Comment:
FYI there is a lot more extraction and test code sharing possible along all four directions of the commiting matrix
| PRU1 | PRU2 |
| ProveCommit / Aggregate | ProveCommit2 |